xEdziu/RPG-Handy-Helper-Archived
# Completed design of login page and pages related to it.

Together with @MANDRW we completed design of login, registrarion, resetting password and forgetting password page. 
Most likely, this page will be changed further in the future, I am aware that media-queries are still missing, but I think that the page is in a condition that allows us to continue working.

How to test it:

1. Start php server(php -S 0.0.0.0:8800 in terminal)
2. Open your browser and go to your localhost address with port provided in step above (127.0.0.1:8800)
3. Check login and registration page.
4. The link to the password reset page is on the login page.

I am waiting for your response and suggestions about visual side of things.
